src/main/java/com/google/sps/data/SaleCard.java

```diff
@@ -23,18 +23,22 @@
   private String businessName;
   private String description;
   private boolean hasDelivery;
+  private boolean isTemporarilyClosed;
+  private boolean openDuringNight;
```

Comment:
Can you clarify why we need this variable? Can't the vendor set their hours, either in the daytime or the nighttime?

Reply:
Yes, they can. There was not a mandatory necessity to have it, I had it as a way to make it easier for pods working on the front (so that they did not need to do any checks), but, as I discussed with them, we decided to remove this variable since it wasn't complicate to replicate the functionality with JS `Date`.